src/controllers/user.controller.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { userService } = require('../services/user.service.js');
+
+const getAll = async(req, res) => {
+  const users = await userService.getAll();
+
+  res.send(users.map(userService.normalizeUser));
+};
+
+const getOne = async(req, res) => {
```

Comment:
Consider using a more descriptive function name like 'getOneUser' instead of 'getOne'